cmd/rt-conf/main.go

```diff
@@ -23,10 +23,14 @@ func main() {
 
 func run(args []string) error {
 	envConfigFile := os.Getenv("CONFIG_FILE")
-	envVerbose, err := strconv.ParseBool(os.Getenv("VERBOSE"))
-	if err != nil {
-		log.Printf("Warning: failed to parse VERBOSE env var, defaulting to false: %v", err)
-		envVerbose = false
+	verboseDefaultCfg := false
+	var err error
+	envVerbose, ok := os.LookupEnv("VERBOSE")
+	if ok {
+		verboseDefaultCfg, err = strconv.ParseBool(envVerbose)
+		if err != nil {
+			return err
```

Comment:
Maybe nicer to chain the error?
```suggestion
			return fmt.Errorf("failed to parse VERBOSE env var: %v", err)
```